### ISSUE TITLE
Haskell-mode debug keys don't work

### ISSUE THREAD

**Issue by user471** [syl20bnr/spacemacs#6069]
#### Reproduction guide

- Start Emacs
- open haskell file
- run debug `, d d`

*Observed behaviour:*

- key `g - refresh` doesn't work. `g` opens `which-key` window

*Expected behaviour:*

- key `g` should work

#### System Info
- OS: windows-nt
- Emacs: 24.5.1
- Spacemacs: 0.105.20
- Spacemacs branch: develop (rev. 121ee00)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-help-tooltip t auto-completion-enable-sort-by-usage t)
 git emacs-lisp haskell colors javascript csharp dash org purescript syntax-checking markdown semantic elm ranger)
```

**Comment by d12frosted** [comment]
You can use `, d r` to call refresh while in `haskell-mode` (not in debug buffer). 

I believe that it's just due to vim editing style. I believe that haskell debug mode should be evilified (or how is it called)?

**Comment by user471** [comment]
>You can use , d r to call refresh while in haskell-mode

Yes, but I think pressing one key, especially for something like step, would be a better option.

>I believe that it's just due to vim editing style. I believe that haskell debug mode should be evilified (or how is it called)? 

Couldn't Spacemacs just ignore specific window and don't interrupt keys on it?

**Comment by d12frosted** [comment]
Well, you are in evil state - that's why it interpreters keys differently.
That's why I think that buffer should be evilified (if I understand what it
means). You can switch to insert state in that buffer and use `g`.

~ d12frosted

**Comment by d12frosted** [comment]
But that makes sense. See #6072. It will allow you to use `g` for refresh (and some other short bindings).

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
